### PR TITLE
ECS current to 8.11

### PR DESCRIPTION
**Pull request by kgeller**
The team will not be releasing an ECS 8.12 while we decide how to approach releases of ECS going forward

I also cleared out some of the older versions from the live options.

___
Relates https://github.com/elastic/security-team/issues/8182

### CODE REVIEW

**Review by kilfoyle — APPROVED** [review]
LGTM!
As long as this passes CI checks it should be good to go. 🚀

**Comment by bmorelli25** [comment]
I'm not convinced this will pass link-checking, but we'll see. After this is merged, we'll need to merge master into @abdonpijpelink's 8.12 PR: https://github.com/elastic/docs/pull/2862

**Comment by kgeller** [comment]
@bmorelli25 Looks like you called it not passing CI. Does that mean it's not possible to omit an ECS version?

**Comment by bmorelli25** [comment]
It doesn't mean it's not possible. It just means it's going to be a much more complex process than this PR. A lot of how we move forward depends on what the future of the ECS release process looks like. Right now, we map ECS docs 1:1 to Stack docs. If that's changing, we'll need to rethink how we link to ECS documentation. I would set some time up to chat with your technical writer about potential next steps.

**Comment by kgeller** [comment]
> A lot of how we move forward depends on what the future of the ECS release process looks like. Right now, we map ECS docs 1:1 to Stack docs. If that's changing, we'll need to rethink how we link to ECS documentation.

Makes sense, and that is still being worked out.

If we hold off on cleaning up the selectors for now, is it possible to keep ECS current at 8.11 for the time being?

**Comment by bmorelli25** [comment]
Hmm. That could potentially work. Wanna edit this PR and see what the checks say? That's the best way to test.

**Comment by kgeller** [comment]
@bmorelli25 looks like CI is happy with ecs current : 8.11

**Review by bmorelli25 — APPROVED** [review]

**Pull request merged**
